src/components/forms/Dropdown/Dropdown.test.tsx

```diff
@@ -1,18 +1,24 @@
 import React from 'react'
 import { render } from '@testing-library/react'
+jest.mock('../../../deprecation')
+import { deprecationWarning } from '../../../deprecation'
 
-import { Select } from '../Select/Select'
+import { Dropdown } from './Dropdown'
 
 describe('Dropdown component', () => {
-  it('renders without errors', () => {
+  it('renders without errors, displaying a deprecation warning', () => {
     const { queryByTestId } = render(
-      <Select id="input-type-text" name="input-type-text">
+      <Dropdown id="input-type-text" name="input-type-text">
         <option>- Select - </option>
         <option value="value1">Option A</option>
         <option value="value2">Option B</option>
         <option value="value3">Option C</option>
-      </Select>
+      </Dropdown>
     )
     expect(queryByTestId('Select')).toBeInTheDocument()
+    expect(deprecationWarning).toHaveBeenCalledTimes(1)
+    expect(deprecationWarning).toHaveBeenCalledWith(
+      'Dropdown is deprecated and will be removed in the future. Please use the Select component instead.'
+    )
```

Comment:
Not really necessary, since deprecationWarning and withDeprecationWarning have tests, but it was peace of mind for me making sure that my suggestions actually work because this was just a theory until confirming it 😉

Reply:
bah i tried adding this but couldn't get it to work